mantidproject/mantid
# Indirect Data Analysis ConvFit - Post plot and save

The plotting and saving of the `_Results` workspace has now been changed to be completed post algorithm execution. This has the advantage of allowing users to change the plot they desire after algorithm completion and not having to rerun the algorithm.

**data available from unit test data directory**
# To test
- Open ConvFit (Interfaces > Indirect > Data analysis > ConvFit) 
- Sample = `irs26176_graphite002_red.nxs`
- Resolution = `irs26173_graphite002_res.nxs`
- Ensure that the `Plot` and `Save` buttons are currently disabled
- Change the Fit type (should test a couple)
- Ensure the `Plot Output` combo box is now populated 
- Run the algorithm using the `Run` button
- Once complete ensure that the `Plot` and `Save` buttons are now enabled
- Try all the `Plot Output` options create plots 
- Ensure `Save` saves the workspace suffixed with `_Result` (this will save to current save dir)
- Delete the `_Result` workspace from the ADS
- Try to use the `Save` / `Plot` buttons 
  - Ensure an appropriate error message is produced in both cases

Fixes #17019

[RELEASE NOTES]()

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
